pokemon-showdown

```diff
@@ -23,13 +23,7 @@ try {
 }
 
 // Start the server
-
-let port;
-if (process.argv[2]) {
-	port = parseInt(process.argv[2]);
-}
-
 // Bootstrap manually to ensure any dependencies that were just installed can
 // be found when running on Windows
```

Comment:
This should also be clearer about how this keeps `app.js`'s `require.main === module`, which is why sockets listen on a passthrough.